app/views/layouts/application.html.slim

```diff
@@ -1,12 +1,11 @@
-- content_for :template, yield(:title) || t(:loans)
-
 - content_for :head
   = javascript_include_tag "application"
   = stylesheet_link_tag "application", :media => "all"
-  = csrf_meta_tags
+  // = csrf_meta_tags
+
+- content_for :template, yield(:title)
```

Comment:
The reason I used `:template` as the title placeholder was so that the page in Wordpress that Rails gets embedded in could be called `[rails_template]` as opposed to the less obvious `[rails_title]`. I don't think there's any reason to keep it that way in the non-embedded system. I would change them to :title, then you can remove this line.